frontend/cypress/.eslintrc.cjs

```diff
@@ -5,6 +5,7 @@ module.exports = {
   plugins: ['cypress'],
   extends: ['plugin:cypress/recommended'],
   rules: {
+    'cypress/no-async-tests': 'off',
```

Comment:
Sorry I should've looked into this last time I flagged this rule. According to the docs this is included in `plugin:cypress/recommended`, so I'm not keen on turning it off. It also doesn't look like we have any async cypress tests, so I think we can safely remove this line.